docs/learn/learn-polkadot-opengov.md

```diff
@@ -530,6 +530,24 @@ Note that to successfully cast votes you need to have the
 [existential deposit](./learn-accounts.md#existential-deposit-and-reaping) and some additional funds
 to pay for transaction fees.
 
+### Voting Locks
+
+A lock is placed on the {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} when you vote on an
+Ongoing referendum. If you choose to remove your vote when the referendum is still in the decision period, 
+your voting lock is removed. Once the referendum is decided (accepted or rejected), your {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} 
+will be available for unlocking if you voted with zero conviction. The governance app or interface you 
+used for participating in Polkadot OpenGov should show an option to unlock your {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }}.
+[Here](https://support.polkadot.network/support/solutions/articles/65000184129) are the instructions to 
+do it on the Polkadot-JS UI.
+
+If you voted with conviction, the corresponding locks start at the end of the Referendum and not at the moment you voted.
+For instance, let's say you voted AYE with 6X conviction on a referendum, and it got accepted; the 
```

Comment:
I think we need to make the difference between voting solo and delegation

Reply:
If you delegate, it does not matter if the tokens were used to vote or not. The lock will start when you undelegate (even if you delegate for 5 min and then you change your mind). If you vote solo and remove the vote before the ref ends -> no lock. If you vote and you are on the losing side -> no lock

Reply:
Good point @filippoweb3 , we should also add that here

Reply:
It is right below this newly created section. I left out these details because we have not introduced delegation yet.

<img width="829" alt="image" src="https://github.com/w3f/polkadot-wiki/assets/86818441/d0463d8d-90e7-4643-9fef-e66c93e84c16">